code/script.js

```diff
@@ -222,104 +306,218 @@ const generateBoard = () => {
   })
 }
 
+
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
-// This function to start (and restart) the game
+// Function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
+  //Reset the counter
+  incrementOne = 0
+  counter.innerText = incrementOne
 }
 
-// setting the currentQuestion object when you select something in the dropdown
+// setting the currentQuestion object when something is selected in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
-
+  const value = questions.options[questions.selectedIndex].value
+  const text = questions.options[questions.selectedIndex].text
   // This variable stores what option group (category) the question belongs to.
   // We also need a variable that stores the actual value of the question we've selected.
-  // const value =
-
+  
   currentQuestion = {
-    category: category,
-    // value: value
+    category: category, 
+    value: value,
+    text: text,
```

Comment:
In the currenQuestion object you have added third property called text, and you use it later to print the text if the secret person has or not has the asked property. Is there a reason you added this third 'text' property and not just used the 'value' when printing out the texts in function filterCharacters?

Reply:
I had the value at first, but it was hard for me to find a good way to print it out in the alerts with "good" English, and also being able to use it as an understandable value inside the characters array. That´s why I decided to save the text as well :)